content/blog/2019/02/2019-02-08-let-s-celebrate-java-11-support.adoc

```diff
@@ -0,0 +1,56 @@
+---
+layout: post
+title: "Let's celebrate Java 11 Support on Jenkins"
+tags:
+- core
+- developer
+- java11
+- community
+- platform-sig
+author: alecharp
+links:
+  gitter: /jenkinsci/platform-sig
+  googlegroup: jenkins-platform-sig
+  sig: platform
+---
+
+NOTE: This is a joint blogpost prepared by the link:https://github.com/orgs/jenkinsci/teams/java11-support[Java 11 Support Team].
```

Comment:
The team is not visible by non-org members. Would be better to add an explicit list of contributors

```suggestion
NOTE: This is a joint blogpost prepared by the Java 11 Support Team: John Doe, Fred Bloggs, Monsieur Durand, etc.
```